.github/workflows/publish.yml

```diff
@@ -0,0 +1,32 @@
+name: Publish
+
+on:
+  push:
+    branches:
+      - main
+      - 1.6.x # remove when we create 'main'
```

Comment:
I want to see the publishing of snapshots to confirm that everything is in place.

Reply:
On call today we decided to not include any release branches. 

`1.6.x` will become `main` and one day `1.7.0.` will be cut from `main` as well. 

If we ever have the need to cut a release for `1.6.x` after `main` becomes `1.7`, then we can create maintenance branches.